functions/vshn-postgres-func/alerting.go

```diff
@@ -30,6 +30,11 @@ func AddUserAlerting(ctx context.Context, iof *runtime.Runtime) runtime.Result {
 		return runtime.NewFatalErr(ctx, "Cannot get composite from function io", err)
 	}
 
+	// Wait for the next reconciliation in case instance namespace is missing
+	if comp.Status.InstanceNamespace == "" {
+		return runtime.NewWarning(ctx, "Composite is missing instance namespace, skipping transformation")
+	}
+
```

Comment:
Nitpick: Maybe add a testcase to test if it actually works

Reply:
Added a unit test